docs/src/pages/index.astro

```diff
@@ -71,7 +74,7 @@ const description =
               title="DiceDB source code on GitHub"
             >
               <span class="icon"><Github /></span>
-              <span>GitHub (7k+)</span>
+              <span>{`GitHub ${stars || '9k'} `}</span>
```

Comment:
Remove this `||` and just use `stars`. Generate website statically and then help me with the screenshot of it to make sure that static generation and population of stars do work.